src/template.rs

```diff
@@ -377,12 +377,18 @@ impl Template {
                     {
                         let (line_no, col_no) = parser.input().line_col(prev_end);
                         if token.rule == Rule::raw_block_end {
-                            let text = &source[prev_end..token.start];
+                            let mut text = &source[prev_end..token.start];
+                            if text.starts_with("\\{{") {
```

Comment:
Good call. I wasn't sure of the grammar only parsed raw text with these at the beginning. I updated the tests, impl, and refactored to a method